src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -640,11 +629,16 @@ public InMoov2Arm getLeftArm() {
 	public InMoov2Hand getLeftHand() {
 		return leftHand;
 	}
+	
+  @Override
+  public String getLanguage() {
+    return Runtime.getInstance().getLanguage();

```

Comment:
This should be the language that the InMoov2 is set to.. not Runtime's language.

Reply:
it is .. should be outdated .. check now